lib/oxidized/model/stoneos.rb

```diff
@@ -0,0 +1,38 @@
+class Stoneos < Oxidized::Model
+#Hillstone Networks StoneOS software
+
+  prompt /^\r?[\w.()-]+[#>](\s)?$/
+  comment '# '
+  
+  expect /^\s.*--More--.*$/ do |data, re|
+   send ' '
```

Comment:
Indentation in this block is inconsistent, 1 character deep instead of 2 (vs rest of functions).